cli/common/result.go

```diff
@@ -0,0 +1,63 @@
+package common
```

Comment:
Could be easily moved into separate patch

Reply:
Yes, I agree. I'm thinking about moving it to separated module `result`, what do you think?

Reply:
Not sure about module. Why is simple file not enough?

Reply:
See #331